alignak/objects/businessimpactmodulation.py

```diff
@@ -76,7 +76,25 @@ def get_name(self):
         :return: business impact modulation name
         :rtype: str
         """
-        return self.business_impact_modulation_name
+        if hasattr(self, 'business_impact_modulation_name'):
+            return self.business_impact_modulation_name
+        return 'Unnamed'
+
+    def is_correct(self):
+        """
+        Check if this object configuration is correct ::
+
+        * Call our parent class is_correct checker
+
+        :return: True if the configuration is correct, otherwise False
+        :rtype: bool
+        """
+
+        # Ok just put None as modulation_period, means 24x7
+        if not hasattr(self, 'modulation_period'):
+            self.modulation_period = None
```

Comment:
It's not the best place to do this, so manage it in __init__ or perhaps if has defaut_fill or something like it (don't remember the function)